vos/vos/vos.py

```diff
@@ -2040,7 +2054,8 @@ def get_node_url(self, uri, method='GET', view=None, limit=None,
                                      next_uri=next_uri,
```

Comment:
This is a repeat of the previous query. I  think, given the structure of VOSpace and the concept of this system that 'fhead' should be implemented as a view in VOSpace.